pkg/test/testenv.go

```diff
@@ -32,9 +33,10 @@ import (
 )
 
 const (
-	clustersToCreate    = 3
-	clusterProtoName    = "cluster-%d"
-	cassOperatorVersion = "v1.9.0"
+	clustersToCreate          = 3
+	clusterProtoName          = "cluster-%d"
+	cassOperatorVersion       = "v1.8.0"
```

Comment:
This should be v1.9.0. It determines the versions of the CassandraDatacenter CRD to install.

Reply:
I think that must have been a regression introduced during rebasing somewhere. My bad.